charts/gitops-server/templates/deployment.yaml

```diff
@@ -63,6 +63,30 @@ spec:
               port: http
           {{- if or .Values.enableLogin .Values.envVars}}
           env:
+            - name: PROFILE_CACHE_LOCATION
+              value: {{ .Values.profileCacheLocation }}
+            - name: HEALTHZ_BIND_ADDRESS
+              value: {{ .Values.watcherHealthzBindAddress }}
+            - name: METRICS_BIND_ADDRESS
+              value: {{ .Values.watcherMetricsBindAddress }}
+            - name: NOTIFICATION_CONTROLLER_ADDRESS
+              value: {{ .Values.notificationControllerAddress }}
+            - name: WATCHER_PORT
+              value: {{ .Values.watcherPort }}
+            - name: OIDC_ISSUER_URL
```

Comment:
I think these `OIDC_*` values should probably be a discreet block in `values.yaml` as they all need to be set together, e.g.:
```yaml
# values.yaml
...
oidc:
  enabled: false
  issuer_url: ''
  client_id: ''
  # etc.
```
These could then be tested for and made conditional, e.g.:
```yaml
```